src/contracts/digital-twin/container.spec.ts

```diff
@@ -100,6 +100,27 @@ describe('Container', function() {
       expect(await container.getContractAddress()).to.match(/0x[0-9a-f]{40}/i);
     });
 
+    it('can create multiple new contracts in parallel without colliding identities', async () => {
+
+      const [container1, container2, container3] = await Promise.all([
+        Container.create(runtimes[owner], defaultConfig),
+        Container.create(runtimes[owner], defaultConfig),
+        Container.create(runtimes[consumer], { accountId: consumer, description, plugin: 'metadata' })
+      ])
+
+      const desc1 = await container1.getDescription();
```

Comment:
In theory this could work to run getDescription() in parallel ... but I'm not sure about our node version ... 

```
      const [desc1, desc2, desc3] = [await container1.getDescription(), await container2.getDescription(), await container3.getDescription()];
```

Reply:
Wait... wouldn't this also be just three single and blocking awaits statements, that would be awaited one by one? oO

So for performance improvements, better go for:

```
const [desc1, desc2, desc3] = await Promise.all([
  container1.getDescription(),
  container2.getDescription(),
  container3.getDescription(),
]);
```